src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,306 @@
+                    }
+                    return;
+                // since this is not really an error, skipping this method
+                case "SynchronizePullRequestAsync":
+                    return;
+                // for all the other methods
+                default:
+                    description.Append("Repository :" + repositoryBranchUpdateHistory.Repository);
+                    break;
+            }
+            description.Append(Environment.NewLine);
+            description.Append("Branch Name : " + repositoryBranchUpdateHistory.Branch);
+            description.Append(Environment.NewLine);
+            description.Append("Error Message : " + repositoryBranchUpdateHistory.ErrorMessage);
+            description.Append(Environment.NewLine);
+            description.Append("Method : " + repositoryBranchUpdateHistory.Method);
+            description.Append(Environment.NewLine);
+            description.Append("Action : " + repositoryBranchUpdateHistory.Action);
+            description.Append(Environment.NewLine);
+            description.Append("/FYI : " + fyiHandles);
+            description.Append(Environment.NewLine + Environment.NewLine + Environment.NewLine);
+
+            // Get repo info used for create/ update gitHub issue
+            Octokit.Repository repo = await client.Repository.Get(_options.Owner, _options.Repository);
+            try
+            {
+                using (ITransaction tx = _stateManager.CreateTransaction())
+                {
+                    if (await gitHubIssueEvaluator.ContainsKeyAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch)))
+                    {
+                        // issue exists so just update the issue
+                        var issueNumber = await gitHubIssueEvaluator.TryGetValueAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch));
+                        Logger.LogInformation("Updating a gitHub issue number : " + issueNumber + " for the error : " + repositoryBranchUpdateHistory.ErrorMessage +
+                            " for the repository : " + repositoryBranchUpdateHistory.Repository);
+                        if (issueNumber.HasValue)
+                        {
+                            try
+                            {
+                                var issue = await client.Issue.Get(_options.Owner, _options.Repository, issueNumber.Value);
+                                var issueUpdate = issue.ToUpdate();
+                                issueUpdate.Body = issue.Body + Environment.NewLine 
+                                    + Environment.NewLine + description.ToString();
+                                issueUpdate.State = ItemState.Open;
+                                await client.Issue.Update(_options.Owner ,_options.Repository, issueNumber.Value, issueUpdate);
+                            }
+                            catch(Exception ex)
```

Comment:
All these "catch" statements that swallow the errors are very dangerous, and generally not super helpful.  One high level handler should be sufficient. We don't need to catch every single exception at every point, especially since we aren't doing anything (except swallowing it and running code below that almost certainly doesn't expect the first half of the method to have silently returned)

Reply:
okay. I will remove the rest of them.

Reply:
I will keep the ones with the transactions and loop. I will remove the try catch for updating the issue and creating the issue.